test/ArrayField_test.js

```diff
@@ -132,7 +132,7 @@ describe("ArrayField", () => {
       const matches = node.querySelectorAll("#custom");
       expect(matches).to.have.length.of(1);
       expect(matches[0].textContent).to.eql(
-        "should NOT have less than 2 items"
+        "should NOT have fewer than 2 items"
```

Comment:
Do you know why these error messages have changed? Is it maybe because of the ajv upgrade?

Reply:
@epicfaace yeah, they changed after ajv upgrade